### PR TITLE
fix: fix a bug in which errors were not displayed while processing was in progress

### LINKED ISSUE #97: Make sure the process doesn't stop.
## Describe the bug

Many processes stop when an error has occurred.
Then, we have to restart the app.

## List of stopping

- A data XML doesn't exist.
- A cache file of a data XML doesn't exist.
- A format of a data XML is wrong.
- ~~Unknown error has occurred in installing and uninstalling.~~

## Solution
Use `try catch` for front-end related parts.

### PR DESCRIPTION
**Pull request by hal-shu-sato**
## Description

Fixes a bug in which errors were not displayed while processing was in progress

## Related Issue

Fixes #97

## Motivation and Context

Reference #97 

## How Has This Been Tested?

OS: Windows 10 Home
Version: 21H1

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by hal-shu-sato** [comment]
@mitosagi I fixed them.

File: src/preload.js [hunk 1]
```diff
@@ -27,16 +27,17 @@ window.addEventListener('DOMContentLoaded', () => {
 });
 
 window.addEventListener('load', () => {
+  const installationPath = document.getElementById('installation-path');
+
   // core
   const checkCoreVersionBtn = document.getElementById('check-core-version');
   checkCoreVersionBtn.addEventListener('click', (event) => {
-    core.checkLatestVersion(checkCoreVersionBtn);
+    core.checkLatestVersion(checkCoreVersionBtn, installationPath);
```
Comment (mitosagi) [inline src/preload.js:35]:
```suggestion
    core.checkLatestVersion(checkCoreVersionBtn, installationPath.value);
```
probably

Comment (hal-shu-sato) [reply src/preload.js:35]: That is true.

**Comment by hal-shu-sato** [comment]
I fixed it.

**Comment by mitosagi** [comment]
It looks good to me.

**Pull request merged**